src/main/java/Duke.java

```diff
@@ -1,10 +1,102 @@
+import java.awt.*;
+import java.util.List;
+import java.util.Scanner;
+import java.util.*;
+
+/* need to parse commands now into todo, deadline, event */
+/* after parsing commands, need to parse the task and the time */
+/* need to have counter that keeps tracks of total tasks in list */
+
 public class Duke {
+
+    private static String getUserInput() {
+        Scanner in = new Scanner(System.in);
+        return in.nextLine();
+    }
+
+    private static void showExitMessage() {
+        System.out.println("\tBye fellow coder! Hope to see you again soon!");
+    }
+
+    private static void listTasks(List<Task> list) {
+        int i = 0;
+        while (i < list.size()) {
+            int num = i+1;
+            System.out.println("\t" + num + ". " + list.get(i).toString());
+            i++;
+        }
+    }
+
+    private static void markAsDone(List<Task> list, int index) {
+        list.get(index).markAsDone();
+        System.out.println("\tNice! I've marked this task as done:");
+        System.out.println("\t\t" + list.get(index).toString());
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
+
+        boolean isOn = true;
+        int taskCounter = 0;
+        List<Task> list = new ArrayList<Task>();
+
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("Hello! I'm Duke");
```

Comment:
Perhaps you can consider putting this under a new method, hello, as this might increase readability?

Reply:
Amended!